src/__tests__/home.test.tsx

```diff
@@ -1,9 +1,7 @@
+import { render } from '@testing-library/react'
```

Comment:
import order should be:-
 import '@testing-library/jest-dom'

import { render, screen } from '@testing-library/react'

import SearchBar from '../components/SearchBar'